sdk/security/keyvault/azadmin/backup/client_test.go

```diff
@@ -21,13 +22,21 @@ import (
 
 func TestBackupRestore(t *testing.T) {
 	client, sasToken := startBackupTest(t)
-
 	testSerde(t, &sasToken)
 
+	// if in live mode, wait 30 seconds before polling again
```

Comment:
Personally I think this code is too clear for a comment, but if you want something here, replace the "30 seconds" stuff with something about "default frequency"

Reply:
comment removed